src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -133,6 +133,48 @@ This infrastructure needs 13 arguments, described hereafter:
 -   **javaOptions** - Java options appended to the command used to
     start the node on the remote host.
 
+-   **deploymentMode** - Specifies how the ProActive node command is started.
+The deploymentMode can take the following values:
+
+*** _autoGenerated_: when this mode is selected the command to start the
+ProActive node will be generated automatically. As a result, the ssh call to
+the nodes will attach the nodes to the infrastructure without modifications to
+the startup command. This mode is selected by default.
+
+*** _useStartupScript_: wtarts the ProActive node using the script in the variable
+`%startupScriptStandard%`, Allowing the user to modify the startup command of the nodes.
+This mode uses the ProActive node agent identified in the `%schedulingPath%`.
+
+
+*** _useNodeJarStartupScript_: enables connecting to the SSHInfrastructureV2
+using node.jar file. This mode uses `%nodeJarUrl%` and `%startupScriptWitNodeJarDownload%`
+variables to generate the startup command.
+
++
+TIP: if deploymentMode field was set to an empty string, the autoGenerated mode will be automatically selected
++
+
+-   **nodeJarUrl** - The full URL path to download the ProActive _node.jar_
+on each node added to the hostList. The URL have to be accessible from the nodes.
+For example, `try.activeeon.com/rest/node.jar`.
+Used only when `useNodeJarStartupScript` is selected.
+
+-   **startupScriptStandard** - Nodes startup script to launch the ProActive nodes using a ProActive node agent.
+The script by default locates java and node agent directory using `%javaPath%` and `%schedulingPath%` variables respectively.
+The user can modify or extend this script to make changes to the node before or after the startup.
+Used only when `useStartupScript` is selected.
+
+-   **startupScriptWitNodeJarDownload** - Nodes startup script to launch the ProActive nodes using _node.jar_.
+To run the ProActive nodes, this script is also expected to download and install the Java Runtime Environment,
+then download and execute ProActive _node.jar_, if they are not already provided by the node.
+It uses `%nodeJarUrl%` variable to get the full URL path for downloading the _node.jar_ file.
+The user can modify or extend this script to make changes to the node before or after the startup.
```

Comment:
`to make changes to the node before or after the startup` -> `to execute commands on the host before or after the Proactive nodes startup.`